packages/form-core/test/validate/Required.test.js

```diff
@@ -0,0 +1,74 @@
+import { expect, fixture, html, unsafeStatic, defineCE } from '@open-wc/testing';
+// import { LitElement } from '@lion/core';
+import { LionField } from '@lion/form-core';
+
+import { Required } from '../../src/validate/validators/Required.js';
+
+/**
+ * @typedef {import('../../types/FormControlMixinTypes.js').FormControlHost} FormControlHost
+ */
+
+/**
+ * @typedef {import('../../types/FormControlMixinTypes.js').HTMLElementWithValue} HTMLElementWithValue
+ */
+
+/** @type {HTMLElementWithValue} */
+let inputNodeTag;
+class RequiredElement extends LionField {
+  connectedCallback() {
+    const inputNode = document.createElement('input');
+    inputNode.slot = 'input';
+    this.appendChild(inputNode);
+    super.connectedCallback();
+  }
+
+  get _inputNode() {
+    return inputNodeTag || super._inputNode;
+  }
+}
+
+const tagString = defineCE(RequiredElement);
+const tag = unsafeStatic(tagString);
+
+describe('Required validation', async () => {
+  const validator = new Required();
+
+  it('get aria-required attribute if element is part of the right tag names', async () => {
+    const el = /** @type {FormControlHost & HTMLElement} */ (await fixture(
+      html`<${tag}></${tag}>`,
+    ));
+
+    Required._compatibleTags.forEach(tagName => {
+      inputNodeTag = /** @type {HTMLElementWithValue} */ (document.createElement(tagName));
+
+      validator.onFormControlConnect(el);
+      expect(el._inputNode).to.have.attribute('aria-required', 'true');
+    });
+
+    // @ts-ignore
```

Comment:
Here it would be nice to add a comment about the fact that we don't add an attr for incompatible tags